File: developerkhm/AndroidStudy/app/src/main/java/com/example/androidstudy/ui/blog/BlogFragment.java
package com.example.androidstudy.ui.blog;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.example.androidstudy.R;

public class BlogFragment extends Fragment {


    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        View root = inflater.inflate(R.layout.fragment_movie, container, false);
        final TextView textView = root.findViewById(R.id.text_home);
        textView.setText("blog");

        return root;
    }
}

File: jjunji/StudyApplication/app/src/main/java/com/example/studyapplication/blog/BlogFragment.kt
package com.example.studyapplication.blog

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.studyapplication.R

class BlogFragment : Fragment() {
    companion object {
        fun newInstance() : BlogFragment {
            return BlogFragment()
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_blog, container, false)
    }
}